specs/order_spec.rb

```diff
@@ -78,33 +79,65 @@
   end
 end
 
-# TODO: change 'xdescribe' to 'describe' to run these tests
-xdescribe "Order Wave 2" do
+
+describe "Order Wave 2" do
   describe "Order.all" do
     it "Returns an array of all orders" do
-      # TODO: Your test code here!
+
+      array = Grocery::Order.all
+
+      array.must_be_instance_of Array
+
```

Comment:
You should also check that the elements of the array are instances of class `Order`. If it gave you a bunch of strings or something like that, that would be a bug!

```ruby
array.must_be_instance_of Array

# Make sure we got the right number of things from the CSV
array.length.must_equal 100

# Make sure everything has been made into an Order
array.each do |order|
  order.must_be_instance_of Grocery::Order
end
```